doc/sphinx-guides/source/developers/dev-environment.rst

```diff
@@ -444,7 +450,7 @@ Log into Minishift and Visit Dataverse in your Browser
 - username: developer
 - password: developer
 
-Visit https://192.168.99.100:8443/console/project/project1/browse/routes and click http://dataverse-glassfish-service-project1.192.168.99.100.nip.io/ or whatever is shows. This assumes you named your project ``project1``.
+Visit https://192.168.99.100:8443/console/project/project1/browse/routes and click http://dataverse-glassfish-service-project1.192.168.99.100.nip.io/ or whatever is shows under "Routes External Traffic". This assumes you named your project ``project1``.
```

Comment:
This says 100 also. I think we should back out of the changes to 102 above and just say that the IP will vary.

Reply:
Oh, duh. It looks like @matthew-a-dunlap was making this IP consistently "100", which is great. I added notes that it can vary in 25cfb30